cinder/templates/etc/_volume-vmware.conf.tpl

```diff
@@ -0,0 +1,16 @@
+{{- define "volume_vmware_conf" -}}
+{{- $context := index . 0 -}}
```

Comment:
It seems we are not using the global $context in this template so we could avoid the tuple foo in this case, no?